Added Mittal and Sung toluene RCM files

morganmayer opened the pull request:
Added 6 toluene RCM files from Mittal and Sung. These were validated with ChemKED version 0.4.1.

morganmayer committed:
Added Mittal and Sung toluene RCM files

bryanwweber commented on Toluene/MittalSung_2007_RCM_Tc_1022K.yaml:
```diff
@@ -0,0 +1,160 @@
+---
+file-authors:
+    - name: Morgan Mayer
```
Can you please indent this by two spaces per level?

bryanwweber replied:
By "this" I mean all of these files

bryanwweber replied:
In the journal article authors, it looks like you have one too many spaces. Otherwise, thanks for fixing this!

bryanwweber commented on Toluene/MittalSung_2007_RCM_Tc_1022K.yaml:
```diff
@@ -0,0 +1,160 @@
+      ignition-type:
+        target: pressure
+        type: d/dt max
+      compression-time:
```
Can you use the new `rcm-data` field here: https://pr-omethe-us.github.io/PyKED/schema-docs.html#rcm-data-keys This also applies to replacing the `volume-history` with `time-histories`. See the example in the documentation: https://pr-omethe-us.github.io/PyKED/ck-tutorial.html#single-data-point-with-volume-history

bryanwweber replied:
That will also let you add fields for Tc and Pc

bryanwweber commented on Toluene/MittalSung_2007_RCM_Tc_1022K.yaml:
```diff
@@ -0,0 +1,160 @@
+datapoints:
+    - temperature:
+        - 297 kelvin
+      ignition-delay:
```
Is there any uncertainty information available for these data?

morganmayer replied:
It looks like there is uncertainty information given. The authors give an initial temperature uncertainty of +/- 0.5 K. Is the initial temperature the temperature specified in the datapoints section? They specify a pressure transducer The authors also state an overall uncertainty of ignition delay is in a range of 7-11%. Would I add a relative uncertainty of 11% to the ignition delay field?

morganmayer replied:
Yes, there is an absolute uncertainty in initial temperature of 0.5 K I will add here. Also, the authors state that ignition delay measurements have an overall uncertainty in the range of 7.5-11%. Should I just add that there is a relative uncertainty of 11% for ignition delay?

bryanwweber replied:
Yeah, probably best to assume worst-case if the authors aren't more specific.

bryanwweber reviewed (changes requested):
Thanks Morgan! A few small comments to address here.

morganmayer committed:
Added temperature uncertainties, fixed spacing, added rcm-data field

morganmayer committed:
added ignition delay uncertainty, fixed author spacing

bryanwweber commented:
Thanks @morganmayer!

morganmayer commented:
Actually, I am now having trouble validating these files with the relative uncertainties now, so I am going to try to fix them. Sorry for the confusion!

morganmayer commented:
@bryanwweber I just realized I missed a couple dashes so I added those in. Also I changed time-histories to time-history and added the quantity field. They all validate now. Should I open a new pull request or revert this one?

bryanwweber commented:
Can you push those changes to this branch? I'm curious to see how GH handles that

morganmayer commented:
I pushed the changes 35 minutes ago. Looks like it only shows up on my fork

bryanwweber commented:
OK then I'll just pull your branch and push the update

morganmayer commented:
@bryanwweber Sounds good. Thanks!

bryanwweber commented:
OK, I force-pushed a new merge commit to master. Thanks for fixing it quickly!
